### ISSUE TITLE
Migrating from MySQL 8 on Ubuntu 22.04 requires extra steps

### ISSUE THREAD

**Issue by choseh** [pretix/pretix#3109]
### Problem and impact

More of a guide than a bug report.

When trying to migrate from MySQL 8(.0.31) pgloader will not be able  to connect to the database, when the pretix user has been created with the default `caching_sha2_password` method.

To successfully start the migration from MySQL 8, I did the following to create a separate user with the "weak" method, just for the migration part:

`create user 'pretixmigration'@'localhost' identified with mysql_native_password by '<somepassword>';`
`grant all privileges on pretix.* to 'pretixmigration'@'localhost';`

in addition you will have to add the following to your mysql config in the [mysqld] block and restart your mysql server:
`default-authentication-plugin = mysql_native_password`

make sure you remove this after the migration process to prevent new users to be created with the weaker password method.

On ubuntu (22.04.1) you will probably also have to build pgloader yourself. Download the sources from github, then
`apt install sbcl` and `make` and use the resulting binary instead of the packaged one (due to this: https://github.com/dimitri/pgloader/issues/1211)

I hope this helps anyone trying to migrate from MySQL 8 on Ubuntu. 

### Expected behaviour

less extra steps :)

**Comment by raphaelm** [comment]
I'm sorry for the bad experience! We tested the guide on Debian, but apparently Ubuntu behaves significantly different, which was a surprise to us. We already added a recommendation to install pgloader from the PostgreSQL apt sources after #3090, which should work on 22.04 as well.

The password issue is interesting, I have not seen caching_sha2_password before, and my Debian stable installation also did not have it as the default. Let's hope this issue helps others and if someone else runs into this, I'm happy to extend the docs.
